src/content/h1/info/readme.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hard-coded Data
+keywords:
+  - color
+  - data
+  - engine
+  - hard-coded
+  - info
+thanks:
+  - to: MosesOfEgypt
+    for: Armor color values
+  - to: Lavadeeto
+    for: Armor color testing and picture
+---
+There are lots of hard-coded values in Halo's engine, such as the multiplayer
+armor color. These values do not typically appear anywhere in tag data, and
+cannot be directly changed or controlled.
+
+# Multiplayer Armor Colors
```

Comment:
When new content gets added it's also good to link to it from some other pages. The `shader_model` tag page might be a good incoming link location since it talks about armour colours a bit already. You can link directly to the armor colours section by using `[link text][slug#heading-in-kebab-case]`.